draft-ietf-quic-transport.md

```diff
@@ -1459,68 +1623,69 @@ To send additional data prior to completing the cryptographic handshake, the
 server then needs to validate that the client owns the address that it claims.
 
 Source address validation is therefore performed during the establishment of a
-connection.  TLS provides the tools that support the feature, but basic
-validation is performed by the core transport protocol.
+connection, by the core transport protocol.
```

Comment:
"performed by the core transport protocol during the establishment of a connection."